### DISCUSSION TITLE
Add function to calculate the total visitors.

### DISCUSSION THREAD

**Discussion by mendax1234** [pradyunsg/furo#492] [Questions] [answered]
Does furo have a function to calculate the total visitors? If not, can it be added? From my perspective, the original visitors badge isn't well integrated with furo theme and can't be shown on every page (l mean the total visitors can be seens on every page).

**Comment by pradyunsg** [comment, marked as answer]
I don't know how you did this in the past, but you can certainly inject an image-based counter, like you did earlier.

This theme does not have any tracking code injected by it, so you'll have to figure this out on your own, based on your past documentation configuration.

**Reply by mendax1234** [reply]
> I don't know how you did this in the past, but you can certainly inject an image-based counter, like you did earlier.
> 
> This theme does not have any tracking code injected by it, so you'll have to figure this out on your own, based on your past documentation configuration.

@pradyunsg Thanks for replying, l haven't done this before, l just have done a badge to calculate the visitors like below.
![image](https://user-images.githubusercontent.com/62934294/182020487-d4735928-7743-44f0-93c6-e0edf47ab8b6.png)

However, l think it is impossible to integrate it on every page. l think to show it on the top of the documentation is good, but it seems that this need to be done in the theme itself, but l don't know how to change this theme. So, would you please help me with it?

Thanks!
